RHEL/6/input/profiles/ftp-server.xml

```diff
@@ -1,5 +1,5 @@
 <Profile id="ftp-server" extends="server">
-<title override="true">ftp</title>
+<title override="true">FTP Server Profile</title>
 <description override="true">This profile is for FTP servers.</description>
```

Comment:
No worries, the more the merrier. @dahaic want to make the additional changes as part of this PR or separately?